Makefile

```diff
@@ -117,6 +117,10 @@ ko:
 ko-publish: ko
 	KO_DOCKER_REPO=${IMAGE_REPO} $(KO) publish --base-import-paths --push=${IMAGE_PUSH} -t ${TAG} ./cmd/operator
 
+ko-deploy: manifests kustomize ko
+	cd config/manager && $(KUSTOMIZE) edit set image controller="ko://github.com/shipwright-io/operator/cmd/operator"
```

Comment:
Is there a reason to prefer to do this instead of just having `ko://...` in the YAML directly? Without this kustomize call, we depend on an image named `controller:latest` (which doesn't exist now on DockerHub, but theoretically _could_ someday...)

Reply:
I can try experimenting with that, would require some tweaks to how kustomize behaves.